apps/graphql/src/types/output/FromToInterface.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { GraphQLInterfaceType } from 'graphql';
+
+import GraphQLRouteStop from './RouteStop';
+import GraphQLBookingType from './BookingType';
+
+export default new GraphQLInterfaceType({
+  name: 'FromToInterface',
```

Comment:
Could you add a description to how `FromToInterface` differs from a Trip, etc.?

Reply:
🚀